### ISSUE TITLE
[RFE] weekday choice

### ISSUE THREAD

**Issue by ins-pirat-ion** [oskutka/rhlp#46]
It would be great to have ability to choose different day of current week.

**Comment by oskutka** [comment]
I agree, that would be a great feature. Not all the restaurants provide that information though.
